python/ray/ml/BUILD

```diff
@@ -335,3 +335,12 @@ py_library(
     name = "ml_lib",
     srcs = glob(["**/*.py"], exclude=["tests/*.py"]),
 )
+
+# Check if all tests have the pytest if __name__... snippet.
```

Comment:
Wondering if there is an easy way to do this for all BUILD files. Otherwise, it's very easy to forget to add this for new BUILD files that are created.

Reply:
I am no bazel expert, but I couldn't really find anything that would allow that. All solutions either involved modifying the BUILD files a little (this PR) or completely